measureimagefocus_README.md

```diff
@@ -0,0 +1,41 @@
+CellProfiler Microscope Image Focus Module Instructions
+Note: the first time it's run, this module needs to download the model/weights, which can take a while.
+
+OS X
+
+From source (recommended)
+
+Install CellProfiler from source
+Using a terminal (found in Applications), install CellProfiler from source by running given commands in command line (instructions)
+Get CellProfiler Plugin. In the terminal, run these commands:
+```
+MYDIR=/tmp/; cd MYDIR
+git clone https://github.com/CellProfiler/CellProfiler-plugins.git  
+cd CellProfiler-plugins/
+git fetch origin pull/11/head:image-quality
+git checkout image-quality
```

Comment:
This line and the line above it won't be necessary when this plugin is merged into master.